README.md

```diff
@@ -1,9 +1,101 @@
-# Northcoders News API
+# News API
+
+Link to hosted project (will land on list of all endpoints):
+https://news-project-r24g.onrender.com/api
+
+##Project Summary
+Welcome to my News API Project. 
+
+This project involves the building of an API so that application data can be accessed programmatically. 
+It mimics the building of a real world backend service, such as Reddit, and provides this information to the front end architecture. 
+All of the available endpoints are in the endpoints.json file. 
+The functionality that has been implemented allows the user to access all of the endpoints with their description, access all topics, articles and users, access comments for a certain article and filter articles by topic.
+The user can also add a comment to an article and delete a comment as well as editing a specific article. 
+
+##Important Information 
+
+###Cloning:
+
+```
+git clone https://github.com/t-adekunle/news-project.git
+```
+
+###Installing Dependencies
+
+####Developer Dependencies:
+Jest:
+```
+npm install --save-dev jest
```

Comment:
Just so you know - as long as these dependencies are listed in the package.json (dev and normal dependencies) - they can just run npm install and they'll get everything they need.